mirar/pipelines/winter/generator.py

```diff
@@ -193,7 +222,7 @@ def winter_photometric_catalog_generator(
 
     if filter_name in ["J", "H"]:
         return Gaia2Mass(
-            min_mag=10,
+            min_mag=0,
```

Comment:
should this be 0? seems bright

Reply:
This is to get all 2MASS stars including bright ones, so that they can be masked in the images in following steps. For individual processors like PhotCal that require fainter sources, these bright sources are rejected in the catalog_purifiers (this necessitated updating catalog_purifiers to work on both science and reference catalogs).